xbmc/utils/FontUtils.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (C) 2005-2022 Team Kodi
```

Comment:
2022-

Reply:
what means?

Reply:
Those headers refer to the actual file not the program itself. So creation date is 2022 not 2005, and you can leave the other empty (==present)

Reply:
I think @enen92 means `Copyright (C) 2022 Team Kodi`. That's what I do for newly created files.